src/app/routes/cpsAsset/getInitialData/augmentWithDisclaimer/index.js

```diff
@@ -11,18 +11,23 @@ const disclaimerBlock = {
   model: {},
 };
 
-const getDisclaimerInsertionPoint = pageData =>
-  getBlocks(pageData).findIndex(block => block.type === 'timestamp') + 1;
+const getDisclaimerInsertionPoint = (pageData, positionFromTimestamp) =>
+  getBlocks(pageData).findIndex(block => block.type === 'timestamp') +
+  positionFromTimestamp;
 
-const insertDisclaimer = pageData =>
+const insertDisclaimer = (pageData, positionFromTimestamp) =>
   setBlocks(
     insert(
-      getDisclaimerInsertionPoint(pageData),
+      getDisclaimerInsertionPoint(pageData, positionFromTimestamp),
       disclaimerBlock,
       getBlocks(pageData),
     ),
     pageData,
   );
 
-export default toggles => pageData =>
-  isDisclaimerToggledOn(toggles) ? insertDisclaimer(pageData) : pageData;
+const augmentWithDisclaimer = (toggles, positionFromTimestamp) => pageData =>
+  isDisclaimerToggledOn(toggles)
+    ? insertDisclaimer(pageData, positionFromTimestamp)
+    : pageData;
+
+export default augmentWithDisclaimer;
```

Comment:
This could have been left as the default export...
```suggestion
export default (toggles, positionFromTimestamp) => pageData =>
  isDisclaimerToggledOn(toggles)
    ? insertDisclaimer(pageData, positionFromTimestamp)
    : pageData;
```

Reply:
Ah cool thanks! I think got in a bit of an export muddle 🫠